cmd/client-nfd/subcmd/root.go

```diff
@@ -0,0 +1,46 @@
+/*
```

Comment:
let's move all `subcmd` folder  to `pkg/client-nfd` same as `nfd-master` or `nfd-worker`

Reply:
As mentioned, the client will include more subcommands so it cannot be adjusted to the `nfd-master` or `nfd-worker` style. Resolving the comment, feel free to open if you want to re-discuss.